### DISCUSSION TITLE
My 3d object will have a broken image like a rocket launch, I can’t find the reason, please help me

### DISCUSSION THREAD

**Discussion by yuchiahui0530** [cvat-ai/cvat#3656] [Q&A] [answered]
Sir
My 3d object will have a broken image like a rocket launch, I can’t find the reason, please help me
The motorcycle rider in the film has irregular broken pictures in the frame(100-120-122-124-126-127), please tell me how to improve this error
I made some 3d videos, and a few videos have similar situations. I tried to use the merge function but the problem could not be solved. Only a few of the objects in the videos have this situation. Please help me

https://cvat.org/tasks/118166/jobs/198940
thank you

**Comment by bsekachev** [comment]
Hi @yuchiahui0530 

I am not sure I correctly understand you problem, but if I try to modify ``broken`` object on frame 100 it became right.
And how did you get these objects?

**Reply by yuchiahui0530** [reply]
Sir, thank you for your reply. This video has a motorcycle rider on pages 100 and 120. She will appear in a rocket-shaped frame. I tried many methods but it still appears. I attach a photo and task file. Please help me solve the problem. ,thanks

**Reply by yuchiahui0530** [reply]
Sir, I don’t know how to upload pictures and files here, so I will send relevant photos and files to your boris.sekachev@intel.com, thank you

**Reply by bsekachev** [reply]
> I don’t know how to upload pictures and files here

Just drag and drop it in the input field when writing a message

**Reply by yuchiahui0530** [reply]
![3D-1](https://user-images.githubusercontent.com/88265703/134352961-3111241a-b838-45ad-bb60-4d0a51566cec.JPG)
![3D-2](https://user-images.githubusercontent.com/88265703/134353011-80ec86a2-0d86-4ba8-b773-f57ac00237ce.JPG)
[task_b0123_n4chn420210429131238-06_backup_2021_09_07_14_39_39.zip](https://github.com/openvinotoolkit/cvat/files/7211067/task_b0123_n4chn420210429131238-06_backup_2021_09_07_14_39_39.zip)

**Comment by bsekachev** [comment, marked as answer]
@yuchiahui0530

Definitely it is a bug. Could you please submit an issue? 
For now as a temporary workaround I can suggest you changing cuboid perspective slightly (on the gif you can see how to do it, just drag the point holding Shift button)
![Peek 2021-09-22 17-00](https://user-images.githubusercontent.com/40690378/134358042-2432c22b-c2db-4324-9471-add127821f94.gif)
y

**Comment by yuchiahui0530** [comment]
Sir, thank you for your reply, I have submit  an issue   now

**Reply by yuchiahui0530** [reply]
Sir,   I  submit an issue yesterday, today the problem  when uploading other 3D videos, is a 3D problem from a different angle to yesterday, thank you

**Reply by bsekachev** [reply]
Yes, I've seen. We will investigate the issue.

**Reply by yuchiahui0530** [reply]
Sir,thank you
